src/quo2/components/dividers/new_messages.cljs

```diff
@@ -0,0 +1,74 @@
+           :purple     {:text-color       colors/purple-50
+                        :background-color colors/purple-50-opa-5}
+           :indigo     {:text-color       colors/indigo-50
+                        :background-color colors/indigo-50-opa-5}
+           :turquoise  {:text-color       colors/turquoise-50
+                        :background-color colors/turquoise-50-opa-5}
+           :blue       {:text-color       colors/blue-50
+                        :background-color colors/blue-50-opa-5}
+           :green      {:text-color       colors/green-50
+                        :background-color colors/green-50-opa-5}
+           :yellow     {:text-color       colors/yellow-50
+                        :background-color colors/yellow-50-opa-5}
+           :orange     {:text-color       colors/orange-50
+                        :background-color colors/orange-50-opa-5}
+           :red        {:text-color       colors/red-50
+                        :background-color colors/red-50-opa-5}
+           :pink       {:text-color       colors/pink-50
+                        :background-color colors/pink-50-opa-5}
+           :brown      {:text-color       colors/brown-50
+                        :background-color colors/brown-50-opa-5}
+           :beige      {:text-color       colors/beige-50
+                        :background-color colors/beige-50-opa-5}}
+   :dark  {:primary    {:text-color       colors/primary-60
+                        :background-color colors/primary-60-opa-5}
+           :purple     {:text-color       colors/purple-60
+                        :background-color colors/purple-60-opa-5}
+           :indigo     {:text-color       colors/indigo-60
+                        :background-color colors/indigo-60-opa-5}
+           :turquoise  {:text-color       colors/turquoise-60
+                        :background-color colors/turquoise-60-opa-5}
+           :blue       {:text-color       colors/blue-60
+                        :background-color colors/blue-60-opa-5}
+           :green      {:text-color       colors/green-60
+                        :background-color colors/green-60-opa-5}
+           :yellow     {:text-color       colors/yellow-60
+                        :background-color colors/yellow-60-opa-5}
+           :orange     {:text-color       colors/orange-60
+                        :background-color colors/orange-60-opa-5}
+           :red        {:text-color       colors/red-60
+                        :background-color colors/red-60-opa-5}
+           :pink       {:text-color       colors/pink-60
+                        :background-color colors/pink-60-opa-5}
+           :brown      {:text-color       colors/brown-60
+                        :background-color colors/brown-60-opa-5}
+           :beige      {:text-color       colors/beige-60
+                        :background-color colors/beige-60-opa-5}}})
+
+(defn new-messages
+  "new-messages params - label, color"
```

Comment:
consider making the label a param to itself.

see discussion from another pr:
https://github.com/status-im/status-mobile/pull/13644#discussion_r954137072

i.e: 
"_No strong opinion, but it will be great if text-value/label is passed as argument, so that all components follow same pattern_"
Ex. [1](https://github.com/status-im/status-mobile/blob/8e3258769e7b1e6b059d403ffe5337871635bea5/src/quo2/components/button.cljs#L109) [2](https://github.com/status-im/status-mobile/blob/8e3258769e7b1e6b059d403ffe5337871635bea5/src/quo2/components/counter.cljs#L22)